content/docs/command-reference/root.md

```diff
@@ -10,11 +10,15 @@ usage: dvc root [-h] [-q | -v]
 
 ## Description
 
-This command returns the path to the root directory of the <abbr>DVC
-project</abbr>, relative to the current working directory. It can be used to
-build a path to a dependency, script, or <abbr>data artifact</abbr>, for
-example. Useful when working in a subdirectory of the project, and needing to
-refer to a file in another directory.
+The `dvc root` command returns the path to the root directory of the <abbr>DVC
+project</abbr>, relative to the current working directory.
+
+Using `dvc root`, you can build a path to a dependency, script, or <abbr>data
```

Comment:
Why were the sentences separated into a paragraph each? I think they're pretty closely related, at least these 2 first ones.

Reply:
Returned to single paragraph format.

Reply:
Thanks. My Q wasn't necessarily rethorical BTW, you're more than welcome to insist on changes if you feel stringly (just FYI). Resolving